app/src/main/java/com/eveningoutpost/dexdrip/GcmActivity.java

```diff
@@ -688,8 +790,25 @@ static void pushCalibration2(double bgValue, String uuid, long offset) {
             GcmActivity.sendMessage(myIdentity(), "cal2", json);
         }
     }
+    public static void pushLibreBlock(String libreBlock) {
+        Log.i(TAG, "libreBlock called: " + libreBlock);
+        if (!Home.get_master()) {
+            return;
+        }
+        if (!JoH.pratelimit("libre-allhouse", 5)) {
```

Comment:
What is the expected frequency of these packets do you think?

Reply:
Well, if the user is continuously connected to the sensor, then probably every 5 minutes.
I guess that if he is near the device and then goes away, and then reconnects it reasonably should be every minute.
The goal was to block storms, so I figured out that 5 seconds should be enough.